src/lib/actions/clickOutside.ts

```diff
@@ -1,15 +1,26 @@
-export const clickOutside = (node: HTMLElement) => {
-	const handleClick = (event: Event) => {
-		if (node && !node.contains(event.target as HTMLElement) && !event.defaultPrevented) {
-			node.dispatchEvent(new CustomEvent('clickoutside'));
-		}
-	};
-
-	document.addEventListener('click', handleClick, true);
-
-	return {
-		destroy() {
-			document.removeEventListener('click', handleClick, true);
-		}
-	};
-};
\ No newline at end of file
+// Copied from svelte-legos
+import {listen} from "svelte/internal";
```

Comment:
note: I find it curious that `prettier` (I suppose) decides to sometimes include spaces around imports, like `import { listen } ...` and sometimes it doesn't.

Could you check if re-running it helps to unify this?

Reply:
Yeah prettier probably hadn't run yet there, should be fixed now and I'm adding a precommit hook that runs it automatically before every commit on coming PR so this won't happen in the future.